WpfSampleDi/App.xaml.cs

```diff
@@ -1,35 +1,39 @@
-﻿using System.IO;
-using System.Windows;
+﻿using System.Windows;
 using Microsoft.Extensions.DependencyInjection;
-using WpfSampleDi.Service;
+using Microsoft.Extensions.Logging;
+using WpfSampleDi.ViewModels;
 
-namespace WpfSampleDi
+namespace WpfSampleDi;
+
+/// <summary>
+/// Interaction logic for App.xaml
+/// </summary>
+public partial class App
 {
-    /// <summary>
-    /// Interaction logic for App.xaml
-    /// </summary>
-    public partial class App : Application
-    {
-        private readonly ServiceProvider _serviceProvider;
+    private readonly ServiceProvider _serviceProvider;
 
-        public App()
-        {
-            var serviceCollection = new ServiceCollection();
-            ConfigureServices(serviceCollection);
+    public App()
+    {
+        var serviceCollection = new ServiceCollection();
+        
+        ConfigureServices(serviceCollection);
 
-            _serviceProvider = serviceCollection.BuildServiceProvider();
-        }
+        _serviceProvider = serviceCollection.BuildServiceProvider();
+    }
 
-        private void ConfigureServices(IServiceCollection services)
-        {
-            services.AddSingleton<ILogBase>(provider => new LogBase(new FileInfo(@"c:\temp\log.txt")));
-            services.AddSingleton<MainWindow>();
-        }
+    private static void ConfigureServices(IServiceCollection services)
+    {
+        services
+            .AddLogging(logging => logging.AddConsole())

```

Comment:
If you want to log to a file, you can use something like [NLog ](https://nlog-project.org/) or [Serilog](https://serilog.net/), they will integrate with standard .NET logging out of the box.